app/scripts/xprizeDemo.xml

```diff
@@ -0,0 +1,217 @@
+<!-- Copyright (C) 2021 Fondazione Istituto Italiano di Tecnologia (IIT) -->
+<!-- All Rights Reserved. -->
+<!-- Author: Giulio Romualdi <giulio.romualdi@iit.it> -->
+
+<application>
+
+  <name>Xrprize Demo (Virtualizer Xsens SenseGlove)</name>
+  <description>Application to run the ANA Avatar XPRIZE demo.</description>
+  <version>1.0</version>
+  <authors>
+    <author email="giulio.romualdi@iit.it">Giulio Romualdi</author>
+  </authors>
+
+  <!-- Modules -->
+  <!-- icub-head -->
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --from camera/ServerGrabberDualDragon.ini --split true</parameters>
+    <node>icub-head</node>
+  </module>
+
+  <module>
+    <name>WalkingModule</name>
+    <node>icub-head</node>
+    <parameters>--from dcm_walking_joint_retargeting.ini</parameters>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters> --device AudioRecorderWrapper --subdevice portaudioRecorder --name /icub/microphone  --min_samples_over_network 4000  --max-samples_over_network 4000  --rate 160000 --samples 16000</parameters>
+    <node>icub-head</node>
+  </module>
+
+  <module>
+    <name>yarpdev</name>
+    <parameters>--device AudioPlayerWrapper --subdevice portaudioPlayer --name /icub/speakers --playback_network_buffer_lenght 0.4</parameters>
```

Comment:
Need to document the parameter renaming in the newer versions of ROS (post November 2020) 
`playback_network_buffer_lenght`  -> `playback_network_buffer_size`.

See https://github.com/robotology/yarp/commit/1734f53a4bbc30755b7819fae2b4be803a3a6bec

Reply:
We will open an issue for this point, and in future when we do update the iCub3 head, we will update this file here.